config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
```

Comment:
build take time to learn about kustomize. it's importent to control it because we can get lost here with all the yamls

Reply:
what is should I change based on this comment? not to deploy to all namespaces?